code-challenges.test.js

```diff
@@ -50,3 +127,16 @@ const hand4 = [7, 2, 7, 2, 7]
 // Expected output: true
 
 // b) Create the function that makes the test pass.
+ const fullcards = (array) => {
+  //created a function name which takes in an array
+         if (hand1[0] === hand1[1] && hand1[2] === hand1[4] );
```

Comment:
I like the direction you are taking here, but there may be a few conditions that this will not pass as it stands.  Sorting the array first can help arrange it so you guarantee a specific order.